test/ui/time-users-id-tasks.js

```diff
@@ -0,0 +1,27 @@
+const puppeteer = require("puppeteer");
+
+(async () => {
+    const uiLocation = "https://nscstrdevusw2thucommon.z5.web.core.windows.net/#/users/6/tasks ";
+    const browser = await puppeteer.launch();   
+    var startDate = new Date();
+    var totalSeconds = 0;     
+    var startTime = startDate.getTime();
+
+    for(var i=0; i<40; i++){
```

Comment:
For mine, I did a global variable like NUM_TIMES = 40, since it won't necessarily be always 40 times.

Reply:
Fixed it thanks!